frontend/src/components/containers/ContainerEditContent.js

```diff
@@ -132,10 +132,13 @@ const ContainerEditContent = ({ token, containerKinds, containersByID, add, upda
           onValuesChange={onValuesChange}
           onFinish={onSubmit}
         >
-          <Item label="Name" {...props("name")} rules={nameRules.concat(requiredRules)}>
+          <Item label="Name" {...props("name")} rules={nameRules.concat(requiredRules)}
+            tooltip="Use [a-z], [A-Z], [0-9], or [ - ][ _ ][ . ]. Space not allowed."
+            extra="Unique name given to a container." >
             <Input />
           </Item>
-          <Item label="Kind" {...props("kind")} rules={requiredRules}>
+          <Item label="Kind" {...props("kind")} rules={requiredRules}
+            extra="Format of the container." >
```

Comment:
Curious to know why you say "Format" here when it could've been "Kind"

Reply:
Defining or explaining a word or an expression using the same expression is not accomplishing much.